src/DurationDatePicker.elm

```diff
@@ -528,7 +530,7 @@ viewCalendarHeader settings viewTime =
                 ]
                 [ div [ id "year" ] [ text year ] ]
             ]
-        , viewWeekHeader settings [ Sun, Mon, Tue, Wed, Thu, Fri, Sat ]
+        , viewWeekHeader settings [ Mon, Tue, Wed, Thu, Fri, Sat, Sun ]
```

Comment:
The single picker uses the week order specified in the settings. Is there a reason we don't do the same here? Seems like this could lead to a disconnect between the settings and the rendered header.